src/views/auth/error.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'default.html' %}
+{% set title = 'Authentication Error' %}
+{% block pageHeading %}Oops!{% endblock %}
+{% block pageContent %}
+{% if errorCode === 'TOO_YOUNG' %}
+<p>Unfortunately, due to legal reasons, only people who are 18 or over on the day of the hackathon can attend :(</p>
+<p>We hope to see you at a later event!</p>
```

Comment:
Could soften it a little more with something like: "But don't let that put you off! We look forward to seeing you at a later event!"